Chat/assignment-chat/server-chat/src/main/java/ru/assignment/net/ServerThread.java

```diff
@@ -0,0 +1,82 @@
+package ru.assignment.net;
+
+import ru.assignment.model.ChatModel;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Created by Андрей on 18.02.2015.
+ */
+public class ServerThread implements Runnable {
+    private int port;
+    private  ChatModel chatModel;
+    private ServerSocket serverSocket;
+    private Map<Integer, Session> sessionMap;
+    int sessionCount;
+
+    public ServerThread(int port) {
```

Comment:
Please pass Server here and use package private access
Don't create ChatModel here, it is better to create it in Main and pass to Server. Add getter to server.